pom.xml

```diff
@@ -26,6 +26,12 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>RELEASE</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
do we need these lines?
let's remove them

Reply:
we need. 
tests failed after removed.

Reply:
because 
![image](https://user-images.githubusercontent.com/34911404/201145081-c100c031-dff8-4bb3-952b-7c443dd3edd4.png)